src/pages/actions-list/actions-list.ts

```diff
@@ -0,0 +1,46 @@
+import { Component } from '@angular/core';
+import { InfiniteScroll, NavController } from 'ionic-angular';
+import { Observable } from 'rxjs';
+
+import Action from '@models/action';
+import ActionsService from '@providers/actions-service/actions-service';
+
+// tslint:disable-next-line:no-unused no-unused-variable
+const LOG_REF = '[ActionsListPage]';
+
+@Component({
+  selector: 'actions-list-page',
+  templateUrl: 'actions-list.html'
+})
+export class ActionsListPage {
+
+  actions: Action[];
+  fetchActionObs: Observable<Action[]>;
+
+  constructor(
+    public navCtrl: NavController,
+    private readonly actionsService: ActionsService
+  ) {
+    this.actions = [];
+    this.fetchActionObs = null;
+  }
+
+  ionViewDidLoad(): void {
+    this.loadMoreActions();
+  }
+
+  loadMoreActions(infiniteScroll?: InfiniteScroll) {
+    const fetchParams = {
+      offset: String(this.actions.length)
+    };
+    this.actionsService.fetchPaginatedActions(fetchParams)
```

Comment:
We should start thinking about this.

```suggestion
    // TODO: handle fetch paginated actions error
    this.actionsService.fetchPaginatedActions(fetchParams)
```